hazelcast/src/test/java/com/hazelcast/test/HazelcastTestSupport.java

```diff
@@ -262,46 +256,8 @@ public static void printAllStackTraces() {
     // ########## logging ##########
     // #############################
 
-    public static void setLoggingLog4j() {
-        System.setProperty("hazelcast.logging.type", "log4j");
-    }
-
-    public static void setLogLevel(Level level) {
-        if (isLog4jLoaded() && logLevel == null) {
-            logLevel = setLogLevelOnAllLoggers(level);
-        }
-    }
-
-    public static void resetLogLevel() {
-        if (isLog4jLoaded() && logLevel != null) {
-            setLogLevelOnAllLoggers(logLevel);
-            logLevel = null;
-        }
-    }
-
-    @SuppressWarnings("unchecked")
-    private static Level setLogLevelOnAllLoggers(Level level) {
-        Logger rootLogger = Logger.getRootLogger();
-        Level oldLevel = rootLogger.getLevel();
-        rootLogger.setLevel(level);
-
-        Enumeration<Logger> currentLoggers = LogManager.getCurrentLoggers();
-        while (currentLoggers.hasMoreElements()) {
-            currentLoggers.nextElement().setLevel(level);
-        }
-
-        return oldLevel;
-    }
-
-    private static boolean isLog4jLoaded() {
-        setLoggingLog4j();
-        try {
-            Class.forName("org.apache.log4j.Logger");
-            Class.forName("org.apache.log4j.Level");
-            return true;
-        } catch (Throwable ignored) {
-            return false;
-        }
+    public static void setLoggingLog4j2() {
```

Comment:
Can't we enable logging for all tests using the new delegating logger? Or is even the INFO level too much output?